bundles/org.openhab.binding.http/src/main/resources/OH-INF/config/config.xml

```diff
@@ -5,13 +5,15 @@
 	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
 
 	<config-description uri="channel-type:http:channel-config">
-		<parameter name="stateTransformation" type="text">
+		<parameter name="stateTransformation" type="text" multiple="true">
 			<label>State Transformation</label>
-			<description>Transformation pattern used when receiving values. Multiple transformation can be chained using "∩".</description>
+			<description>Transformation pattern used when receiving values. Multiple transformations can be chained by listing
+				each transformation in a separate line, or by concatenating them with "∩".</description>
```

Comment:
In `README.md` you use `listing ON a separate line`.  Is it intentional to use `listing IN a separate line` here?

Reply:
I've updated them to "on a separate line" but my grammar isn't the best.